static/js/rest.js

```diff
@@ -73,21 +75,24 @@ export const makeReducer = (endpoint: Endpoint, initialState: RestState = INITIA
   };
 };
 
-export function makeFetchFunc(endpoint: Endpoint): (...args: any) => Promise<*> {
-  return (...args) => fetchJSONWithCSRF(endpoint.url, endpoint.makeOptions(...args));
+export function makeFetchFunc(endpoint: Endpoint): (params?: Object) => Promise<*> {
+  return (params = {}) => {
+    let url = endpoint.urlTemplate ? endpoint.urlTemplate(params) : endpoint.url;
+    return fetchJSONWithCSRF(url || '', endpoint.fetchOptions(params));
```

Comment:
@aliceriot sorry, had to revert back to the `url`/`urlTemplate` approach to make flow happy, and now i did this (`url || ''`) for the same reason. i dont like it, but it gets flow to STFU. let me know if you have any better suggestions